cypress/support/index.js

```diff
@@ -0,0 +1,52 @@
+import './commands';
+
+// Initialize console and XHR monitoring
+before(() => {
+  let polyfill;
+  // Polyfills must be in umd format - cypress will coerce (and fail) to load this as a fixture or ES/CJS module
+  cy
+    .log('Polyfill Fetch >>> XHR Fallback')
+    .readFile('node_modules/unfetch/dist/unfetch.umd.js')
+    .then(file => polyfill = file);
+  Cypress.on('window:before:load', (win) => {
+    // Spy on console output - Disabled for now (so many background routing errs...)
+    // cy.spy(win.console, 'log')
+    // cy.spy(win.console, 'error')
+    // Cy does not support fetch - make it fallback to XHR, which is supported by Cy
+    delete win.fetch;
+    win.eval(polyfill);
+    win.fetch = win.unfetch;
+  });
+});
+
+// Initialize generic tracking for in-flight XHR
+beforeEach(function () { // May need to reference `this` in the future
```

Comment:
This loop initializes a server that'll spy on XHR and increment/decriment an internal cypress value when XHR is in flight.